api/src/main/java/io/opencensus/internal/OpenCensusThreadFactory.java

```diff
@@ -14,14 +14,16 @@
  * limitations under the License.
  */
 
-package io.opencensus.implcore.internal;
+package io.opencensus.internal;
```

Comment:
Thanks for the comment!

I agree. opencensus-api is not a good place for this utility class since technically users may decide to keep an older version of opencensus-api (e.g. for compatibility concern) and in such case the exporter will fail.

I can live with duplicating the logic in stack exporter. Actually I think to some extent we've already done such duplication on the worker thread logic.